tutorial/readme.rst

```diff
@@ -5,21 +5,21 @@ The Great PyNE Tutorial
 
 **Contents:**
 
-1. `Intro <http://nbviewer.ipython.org/urls/raw.github.com/pyne/tutorial/master/00-intro.ipynb>`_
-2. `Nuclide Naming <http://nbviewer.ipython.org/urls/raw.github.com/pyne/tutorial/master/01-nuclide-naming.ipynb>`_
-3. `Materials <http://nbviewer.ipython.org/urls/raw.github.com/pyne/tutorial/master/02-materials.ipynb>`_
-4. `Nuclear Data <http://nbviewer.ipython.org/urls/raw.github.com/pyne/tutorial/master/03-nuclear-data.ipynb>`_
-5. `Reaction Names <http://nbviewer.ipython.org/urls/raw.github.com/pyne/tutorial/master/04-reaction_names.ipynb>`_
-6. `Cross Sections <http://nbviewer.ipython.org/urls/raw.github.com/pyne/tutorial/master/05-cross-sections.ipynb>`_
-7. `Transmutation <http://nbviewer.ipython.org/urls/raw.github.com/pyne/tutorial/master/06-transmutation.ipynb>`_
-8. `Mesh Basics <http://nbviewer.ipython.org/urls/raw.github.com/pyne/tutorial/master/07-mesh-basics.ipynb>`_
-9. `Diffusion <http://nbviewer.ipython.org/urls/raw.github.com/pyne/tutorial/master/08-diffusion.ipynb>`_
-10. `Outro <http://nbviewer.ipython.org/urls/raw.github.com/pyne/tutorial/master/09-outro.ipynb>`_
+1. `Intro <http://nbviewer.jupyter.org/urls/raw.github.com/pyne/tutorial/master/00-intro.ipynb>`_
```

Comment:
link that actually opens is:
https://nbviewer.jupyter.org/github/pyne/tutorial/blob/master/*

Should I change these to that?

Reply:
If that works to open the notebooks, then I think it's good to change to that.  It's possible that this file was originally written at a time when Github didn't render notebooks correctly/automatically

Reply:
Great, will do